activation/handler.go

```
@@ -405,14 +385,16 @@ func (h *Handler) handleAtxData(ctx context.Context, data []byte) error {
 	if err != nil {
 		return fmt.Errorf("received syntactically invalid atx %v: %v", atx.ShortString(), err)
 	}
-
+	if !atx.Verified() {
```

this check here is out of place for me. if it's not verified, `SyntacticallyValidateAtx` should return an error, right?

---

removed